src/plugins/intel_cpu/src/nodes/experimental_detectron_roifeatureextractor.cpp

```diff
@@ -346,66 +183,245 @@ void ExperimentalDetectronROIFeatureExtractor::initSupportedPrimitiveDescriptors
     if (!supportedPrimitiveDescriptors.empty())
         return;
 
+    Precision inputPrec = getOriginalInputPrecisionAtPort(INPUT_FEATURES_START);
+    Precision outputPrec = getOriginalOutputPrecisionAtPort(OUTPUT_ROI_FEATURES);
+
+    if (inputPrec != Precision::FP32 || outputPrec != Precision::FP32) {
+        if ((outputPrec == Precision::BF16 || inputPrec == Precision::BF16) && mayiuse(avx512_core)) {
+            outputPrec = inputPrec = Precision::BF16;
+        } else {
+            outputPrec = inputPrec = Precision::FP32;
+        }
+    }
+
     std::vector<PortConfigurator> inDataConf;
     inDataConf.reserve(inputShapes.size());
-    for (int i = 0; i < inputShapes.size(); ++i)
-        inDataConf.emplace_back(LayoutType::ncsp, Precision::FP32);
+    inDataConf.emplace_back(LayoutType::ncsp, Precision::FP32);
+    for (int i = 1; i < inputShapes.size(); ++i)
+        inDataConf.emplace_back(LayoutType::ncsp, inputPrec);
+
+    impl_desc_type impl_type;
+    if (mayiuse(cpu::x64::avx512_core)) {
+        impl_type = impl_desc_type::jit_avx512;
+    } else if (mayiuse(cpu::x64::avx2)) {
+        impl_type = impl_desc_type::jit_avx2;
+    } else if (mayiuse(cpu::x64::sse41)) {
+        impl_type = impl_desc_type::jit_sse42;
+    } else {
+        impl_type = impl_desc_type::ref;
+    }
 
     addSupportedPrimDesc(inDataConf,
-                         {{LayoutType::ncsp, Precision::FP32},
+                         {{LayoutType::ncsp, outputPrec},
                           {LayoutType::ncsp, Precision::FP32}},
-                         impl_desc_type::ref_any);
+                         impl_type);
+}
+
+void ExperimentalDetectronROIFeatureExtractor::createPrimitive() {
+    Node::createPrimitive();
```

Comment:
Seems like not needed at all to call base one, just introduce additional overhead. can be removed.